File: backend/src/basico/002-ejercicios/001-mayor_de_dos_numeros/higher_number.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

# Asignación múltiple
num1, num2 = (3, 5) 

def mayor(a, b):
    if a >= b:
        return a
    else:
        return b

print(mayor(num1, num2))

'''
Más fácil:

max([num1, num2])
'''

File: backend/src/basico/002-ejercicios/001-mayor_de_dos_numeros/higher_number.rb
#!/usr/bin/ruby

num1, num2 = 3, 5

# Ruby es muy simple!

#def mayor(a, b)  <----- Así también funciona
def mayor a, b
    if a > b    # Así tambien ----> if (a > b)
        a
    else
        b
    end
end

puts mayor num1, num2
#puts mayor(num1, num2) <----- Así también

